internal/kms/secretskms.go

```diff
@@ -49,7 +49,7 @@ const (
 )
 
 // SecretsKMS is default KMS implementation that means no KMS is in use.
```

Comment:
```suggestion
// secretsKMS is default KMS implementation that means no KMS is in use.
```
change the caps in struct comment too, other places too

Reply:
@Rakshith-R done